flipperdevices/Flipper-Android-App
# Fix bytes race

**Background**

When sending a lot of bytes to flipper, some of them may be missed. This is happening because of a bug inside new `FlipperSerialOverflowThrottler`

**Changes**

- Fix receive for bytearray
- Fix long-polling request

**Test plan**

- Open sample (probably a 10 times to see that flipper now connecting normally)
- Open file manager and upload at least >1024KiB-sized file
- (or) open some key in text editor and save it as another file
- See uploaded/saved file has the same size as original
